src/Repository/ReferentTrait.php

```diff
@@ -12,4 +15,46 @@ private function checkReferent(Adherent $referent): void
             throw new \InvalidArgumentException('Adherent must be a referent.');
         }
     }
+
+    protected function aggregateCountByMonth(array $itemsCount, string $type, int $months = 6): array
```

Comment:
These 2 new methods are not really related to Referent stuff.
I don't have a lot of idea : We keep these like this, we move them to static class, we move them to a properly named trait.

Reply:
I'm agree. This trait contains different functions used in many repository classes. What about renaming this trait to `RepositoryTrait` or transforming to `RepositoryUtils` static class?